model_analyzer/entrypoint.py

```diff
@@ -17,22 +17,17 @@
 from .cli.cli import CLI
 from .model_analyzer_exceptions import TritonModelAnalyzerException
 from model_analyzer.constants import LOGGER_NAME
-from model_analyzer.constants import CONFIG_PARSER_FAILURE
-from .triton.server.server_factory import TritonServerFactory
-from .triton.server.server_config import TritonServerConfig
+from .triton.server.server_handler import TritonServerHandler
 from .triton.client.client_factory import TritonClientFactory
 from .state.analyzer_state_manager import AnalyzerStateManager
 from .config.input.config_command_profile import ConfigCommandProfile
 from .config.input.config_command_analyze import ConfigCommandAnalyze
 from .config.input.config_command_report import ConfigCommandReport
 from .log_formatter import setup_logging
-from model_analyzer.config.input.config_utils import \
```

Comment:
I love seeing so much deleted code!! None of this should have ever been here, and I'm glad it is being moved to a more appropriate place!

Reply:
Thanks!